MovieJournalMVC/View/MovieReviewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MovieReviewCell.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import UIKit
+
+class MovieReviewCell: UITableViewCell {
+
+    
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    var title: String?
+    var review: String?
+    
+    
+    
+    func getCellText(entry: String) {
+        self.textLabel?.text = entry
```

Comment:
This can simply be `textLabel.text = entry`, though you may want a better name for your label, especially if you add more than one.

Reply:
The label is now movieTitleLabel.